tests/EmailReplyParser/Tests/Parser/EmailParserTest.php

```diff
@@ -410,6 +410,24 @@ public function testEmailWithFairAmountOfContent()
         $this->assertRegexp('/^On Thursday/', (string) $fragments[0]);
     }
 
+    /**
+     * override regexp, not to match too greedy signature.
+     *
+     * @see https://github.com/willdurand/EmailReplyParser/pull/42
```

Comment:
I would simply add this:

```
// See: https://github.com/willdurand/EmailReplyParser/pull/42.
```

Reply:
`@see` is official `@phpdoc` annotation, phpstorm didn't make links clickable in the past. seems that's not the case anymore. can drop.

http://docs.phpdoc.org/references/phpdoc/tags/see.html